### ISSUE TITLE
Cmake error: unknown policy

### ISSUE THREAD

**Issue by danpla** [slembcke/Chipmunk2D#112]
Trying to compile Chipmunk on Linux with Cmake 2.8.12.2 gives an error:
`Policy "CMP0042" is not known to this version of CMake.`

It's because this policy [was introduced in Cmake 3.0](https://cmake.org/cmake/help/v3.0/policy/CMP0042.html).

It must be:

```
if(POLICY CMP0042)
  cmake_policy(SET CMP0042 NEW) # use MACOSX_RPATH
endif()
```

**Comment by BlaXpirit** [comment]
This problem is still present.

**Comment by danpla** [comment]
Partially fixed in d2883f3edf95d329fea96ddf2d7e156afa4180fe, but will still fail with CMake < 3.0 on OS X.
